src/api/app/views/webui/groups/_group_members.html.haml

```diff
@@ -10,7 +10,8 @@
         .card.m-1.p-2.group-user
           - if write_access
             = render(partial: 'webui/groups/group_members_delete_dialog', locals: { user: user, group: group })
-            = link_to('#', data: { toggle: 'modal', target: "#delete-group-members-modal-#{user}" }, title: 'Remove member from group') do
+            = link_to('#', data: { toggle: 'modal', target: "#delete-group-members-modal-#{user.to_s.gsub('.', '\\\.')}" },
```

Comment:
There is [`dom_id`](https://api.rubyonrails.org/classes/ActionView/RecordIdentifier.html#method-i-dom_id) for that purpose. We should not use the login for referral.

Reply:
True, I didn't think about that.